app/Http/Controllers/ScheduleController.php

```diff
@@ -148,184 +149,94 @@ public function destroy($id)
             return Redirect::back();
         }
         // Delete all corresponding entries first because of dependencies in database
-        foreach ( $schedule->getEntries()->get() as $entry ) {
-            $result = (new ScheduleEntryController)->destroy($entry->id);
+        foreach ( $schedule->shifts->get() as $shift ) {
+            $result = (new ShiftController)->destroy($shift->id);
```

Comment:
This was just a rename all entries to shifts. Should probably also put this in the ShiftController.